packages/hop-node/src/watchers/BondWithdrawalWatcher.ts

```diff
@@ -257,11 +278,15 @@ class BondWithdrawalWatcher extends BaseWatcher {
           bondWithdrawalAttemptedAt: 0
         })
       }
+      if (err instanceof PreTransactionValidationError) {
+        logger.error('pre transaction validation error. turning off writes.')
+        globalConfig.emergencyDryMode = true
```

Comment:
suggestion: use setter function for setting global config values

```suggestion
        enableEmergencyDryMode()
```

in config.ts

```ts
export function enableEmergencyMode() {
 config.emergencyDryMode = true
}
```